aem/aem_skyline_install.md

```diff
@@ -0,0 +1,98 @@
+
+## Integrate with AEM as a Cloud Service
+
+This documentation is specific to `AEM as a Cloud Service` set up, to integrate with `AEM on premise` 
+please refer to the other associated [documentation](aem_on_premise_install.md).
+
+### Pre-release provisioning
+
+If you are an existing `AEM as a Cloud Service` customer, please contact your Adobe representative providing your AEM authors public urls.
+
+For that, log-in to [your cloud manager](my.cloudmanager.adobe.com/) space, look up your authors environments
+
+  ![Cloud Manager Environments](../img/cloud_manager_environments.png "Cloud Manager Environments")
+
+Select from this list the environments public url you want Adobe I/O to expose as `Events Providers` . 
+
+As soon as the Adobe I/O team is done provisioning it, your new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console.
+ 
+  ![An Adobe I/O Console `AEM as a Cloud Service` Events Provider](../img/add_skyline_event_provider.png "An Adobe I/O Console `AEM as a Cloud Service` Events Provider")
```

Comment:
Also, I am not sure if this is easily understood by the outside world. Should we add more screenshots on how to access this provider from console?

Reply:
I added a link to a related doc